pkg/operator/ceph/csi/secrets.go

```diff
@@ -81,6 +81,7 @@ func createCSIKeyringCephFSProvisioner(s *keyring.SecretStore) (string, error) {
 func cephCSIKeyringRBDNodeCaps() []string {
 	return []string{
 		"mon", "profile rbd",
+		"mgr", "allow rw",
```

Comment:
If an upgraded cluster, will the new caps be applied?

Reply:
Yes @travisn that is handled.